indico/util/date_time.py

```diff
@@ -27,14 +27,36 @@
 from babel.dates import get_timezone
 from babel.numbers import format_number as _format_number
 from dateutil.rrule import rrule, DAILY, MO, TU, WE, TH, FR, SA, SU
-from dateutil.relativedelta import relativedelta
+from dateutil.relativedelta import relativedelta as _relativedelta
 
 from indico.core.config import Config
 from indico.util.i18n import get_current_locale, _, ngettext, parse_locale
 from indico.util.string import inject_unicode_debug
 from indico.legacy.common.timezoneUtils import DisplayTZ
 
 
+class relativedelta(_relativedelta):
+    """Improved `relativedelta`"""
```

Comment:
With `__abs__`. See the [ongoing discussion here](https://github.com/dateutil/dateutil/issues/350#issuecomment-284470151).